subjects/java/checkpoints/builder-blueprint/README.md

```diff
@@ -0,0 +1,96 @@
+## Builder Blueprint
+
+### Instructions
```

Comment:
@zanninso It's not at all clear to me what it is supposed to do in this exercise: could you please elaborate a bit more for me? :sweat_smile:

Reply:
i guess I explained that enough in Discord, If you need more clarification i will be happy to explain to you here or even doing a call